tests/integration/file.rs

```diff
@@ -223,6 +223,31 @@ fn test_file_std_read() {
     assert_eq!(buf, "hello, world!".as_bytes());
 }
 
+/// Test `std::io::Seek` impl.
+#[cfg(feature = "std")]
+#[test]
+fn test_file_std_seek() {
+    let fs = load_test_disk1();
+    let mut file = fs.open("/small_file").unwrap();
+
+    let mut buf = [0; 13];
+
+    // Seek from start.
+    assert_eq!(file.seek(SeekFrom::Start(1)).unwrap(), 1);
+    assert_eq!(file.read(&mut buf).unwrap(), 12);
+    assert_eq!(buf, "ello, world!\0".as_bytes());
+
+    // Seek from end.
+    assert_eq!(file.seek(SeekFrom::End(-6)).unwrap(), 7);
+    assert_eq!(file.read(&mut buf).unwrap(), 6);
+    assert_eq!(&buf[..6], "world!".as_bytes());
+
+    // Seek from current position.
+    assert_eq!(file.seek(SeekFrom::Current(-6)).unwrap(), 7);
+    assert_eq!(file.read(&mut buf).unwrap(), 6);
+    assert_eq!(&buf[..6], "world!".as_bytes());
+}
```

Comment:
Why no tests producing `InvalidInput`? e.g. testing something resulting in a negative `pos` (i think?)

Reply:
Added